lib/src/execute.rs

```diff
@@ -276,11 +277,15 @@ impl ExecuteCtx {
             .expect("`memory` is exported")
             .size(&store);
 
+        let distant = Instant::now().duration_since(now);
+
         info!(
             "request completed using {} of WebAssembly heap",
             bytesize::ByteSize::kib(heap_pages as u64 * 64)
         );
 
+        info!("request completed in {:?}", distant);
```

Comment:
same rename as above, just applying it here so you can fix this with the click of a mouse :computer_mouse: 

```suggestion
        info!("request completed in {:?}", request_duration);
```